src/terminal/adapter/ut_adapter/adapterTest.cpp

```diff
@@ -1665,6 +1684,40 @@ class AdapterTest
         _testGetSet->ValidateInputEvent(L"\033P0$r\033\\");
     }
 
+    TEST_METHOD(RequestModeTests)
+    {
+        BEGIN_TEST_METHOD_PROPERTIES()
+            TEST_METHOD_PROPERTY(L"Data:modeNumber", L"{1, 3, 5, 6, 8, 12, 25, 40, 66, 67, 1000, 1002, 1003, 1004, 1005, 1006, 1007, 1049, 9001}")

```

Comment:
Ugh, I wish there was an easy way we could just iterate over `ModeParams`. Should we at least mention that the supported modes are stored in DispatchTypes.hpp as `ModeParams`?

Reply:
Yeah, I've added a comment there now. And thanks for catching all those typos.